build.ps1

```diff
@@ -11,6 +11,8 @@ param (
     [string]$NuGetPFXPath,
     [switch]$SkipXProj,
     [switch]$SkipCSProj,
```

Comment:
SkipCSProj will still skip the building of CS projects if you want to run tests only. ./build.ps1 -SkipXProj -SkipCSProj -SkipDev15 would run client tests on Dev14 without building, for example.